src/main/java/gov/usds/case_issues/validators/TagFragment.java

```diff
@@ -16,15 +16,15 @@
 /**
  * A custom constraint for "one word" validations, to avoid repeating (or nearly repeating)
  * the same {@link Pattern} annotation in a million places.
- */
-@Constraint(validatedBy = { })
-/**
- * \\w matchs 0 or more word chars [A-Z, a-z, 0-9, _]
- * [-\\w]* match 0 or more words seperated by [-]
+ * <ul>
+ * <li>\\w* matches 0 or more word chars [A-Z, a-z, 0-9, _]
```

Comment:
Do the list item elements need to be closed (`</li>`)?

Reply:
As far as I know, javadoc formatting assumes approximately HTML 2.0, so it works fine without (to the extent that I tested this out, it worked, but that was just checking if Eclipse generated the appropriate hover text).